### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (84)

This is the login page of my E-commerce website. Please review it!

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean . Everything looks good to me. You are good to go.

File: Pages/signup.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://ozone-ui-library.netlify.app/styles.css">
+    <link rel="stylesheet" href="../css/navbar.css">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="../css/root.css">
+    <link rel="stylesheet" href="../css/signup-login.css">
+    <title>Spots Cave | Signup</title>
```
Comment (zoyatanweer) [inline Pages/signup.html]: it should be sports cave not "spots cave"

File: css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.right-nav {
+    height: 0rem;
+    align-content: center;
+}
+
+.btn {
+    margin: 10px;
+    padding: 0.7rem;
```
Comment (zoyatanweer) [inline css/navbar.css:45]: this can be in pixels.

File: css/root.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    font-size: 1rem;
+}
+
+.justify-align {
+    justify-content: center;
+    align-items: center;
+}
+
+.title {
+    margin: 0.625rem 0rem 1.25rem 0.625rem;
```
Comment (zoyatanweer) [inline css/root.css:23]: change the 0.625 rem value to pixels

File: css/signup-login.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.form-container {
+    width: 25%;
+    height: 65%;
+}
+
+.input-box {
+    width: 100%;
+    margin: 0.625rem 0rem;
```
Comment (zoyatanweer) [inline css/signup-login.css:19]: use px

File: index.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://ozone-ui-library.netlify.app/styles.css">
+    <link rel="stylesheet" href="./css/navbar.css">
+    <link rel="stylesheet" href="./css/root.css">
+    <link rel="stylesheet" href="./style.css">
+    <title>Sports Cave</title>
```
Comment (zoyatanweer) [inline index.html:15]: change to sports cave

**Review by zoyatanweer — APPROVED** [review]
looks good to merge with suggested changes

**Pull request merged**